Bottom: 0ca891b10a83db7e166326df512728ae95c3acf4
Top:    52dec1d3bd5c82a5342ca6fee6abd7904e41865e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-24 17:01:08 +0100

Refresh of move-more-gdk-specific-drawing

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 7686106..58136b6 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -507,9 +507,10 @@ void ghid_draw_rect (hidGC gc, int x1, int y1, int x2, int y2);
 void ghid_fill_circle (hidGC gc, int cx, int cy, int radius);
 void ghid_fill_polygon (hidGC gc, int n_coords, int *x, int *y);
 void ghid_fill_rect (hidGC gc, int x1, int y1, int x2, int y2);
+void ghid_invalidate_lr ();
+void ghid_invalidate_all ();
 
 /* gtkhid-main.c */
-void ghid_invalidate_all ();
 void ghid_get_coords (const char *msg, int *x, int *y);
 gint PCBChanged (int argc, char **argv, int x, int y);
